README.md

```diff
@@ -165,12 +165,9 @@ container to the docker group for socket permissions.
 
 ### `token`
 
-[Generate a Personal Access Token (classic)](https://github.com/settings/tokens), with the `repo:public_repo` scope for only public repositories or the `repo` scope for public and private repositories, and add it to _Secrets_ (repository settings) as `RENOVATE_TOKEN`.
+[Generate a GitHub Personal Access Token (fine-grained is recommended)](https://github.com/settings/tokens) (see the [GitHub authentication docs](https://docs.renovatebot.com/modules/platform/github/#authentication)) and add it to _Secrets_ (repository settings) as `RENOVATE_TOKEN`.
 You can also create a token without a specific scope, which gives read-only access to public repositories, for testing.
-This token is only used by Renovate, see the [token configuration](https://docs.renovatebot.com/self-hosted-configuration/#token), and gives it access to the repositories.
-The name of the secret can be anything as long as it matches the argument given to the `token` option.
-
-Note that Renovate _cannot_ currently use [Fine-grained Personal Access Tokens](https://github.com/settings/tokens?type=beta) since they do not support the GitHub GraphQL API, yet.
+The secret's name can be anything if it matches the argument given to the `token` option.
```

Comment:
What does this mean?

Reply:
That line was there before 🤷 .

My understanding of it is that, when using this GitHub Action, you can call the GitHub Secret however you want, as long as you pass it to the `token` input. E.g.:

```yaml
      - name: Self-hosted Renovate
        uses: renovatebot/github-action@v41.0.12
        with:
          docker-user: root
          token: ${{ secrets.MY_SUPER_RENOVATE_TOKEN_WITH_MY_OWN_NAME_YAY }}
```